code/_core/obj/projectile/magic.dm

```diff
@@ -2,6 +2,77 @@
 	icon = 'icons/obj/projectiles/magic.dmi'
 
 	collision_bullet_flags = FLAG_COLLISION_BULLET_LIGHT
+	//Do go boom?
+	var/does_explode = FALSE
+	var/explode_power = 5
```

Comment:
These two vars can be combined, honestly. Just use explode_power and check if explode_power > 0 for does_explode usage

Reply:
combined